### ISSUE TITLE
SubItems feature status?

### ISSUE THREAD

**Issue by Alexwijn** [Prototik/HoloEverywhere#796]
Some while ago I noticed the SubItems code and asked about it. I have been told that it will be implemented. But I'm wondering at what timeframe it can be realized at the moment?

**Comment by Prototik** [comment]
At least 70%... I'll try to finish impl in couple days.

**Comment by Alexwijn** [comment]
Ok, thanks. I'm looking forward to it.

**Comment by Prototik** [comment]
I'm a lazy bastardo.
![screenshot from 2014-03-03 16 08 41](https://f.cloud.github.com/assets/1416030/2307486/2cb2ef26-a2ab-11e3-8293-51a75c7a8e83.png)

But I'll write it! :)

**Comment by Alexwijn** [comment]
Looks great! I assume the subitems are layouts that can be adjust like their parents.

**Comment by Prototik** [comment]
When I said basic - I mean really _basic_... I have not enough time for make it quickly, sorry.
![screenshot from 2014-03-18 20 01 19](https://f.cloud.github.com/assets/1416030/2447287/1123025a-ae95-11e3-8e94-e492abb79334.png)

**Comment by Alexwijn** [comment]
I just attempt to update to the master version, replaced all files. But I get the following exception: `Couldn't bind menu on view.`

I guess the view he is talking about is menu.xml (http://pastebin.com/8fNnpaEE). That will be called from my code: `SliderMenu sliderMenu = addonSlider().obtainDefaultSliderMenu(R.layout.menu);`. Is there anything I need to change to match the new version?

**Comment by Prototik** [comment]
@Alexwijn yep, listview's id should be `@+id/slider_menu`, not `@android:id/list`

**Comment by Alexwijn** [comment]
Thanks. It opens now without any errors about it.

Could you give me a code snippet? Since I'm unable to add any subitems to an existing item.
`SliderItem categories = sliderMenu.add(R.string.betting_categories, BetsFragment.class, background);
categories.add(categoriesText[0], FootballFragment.class, background);`

I'm not receiving any errors but there is no arrow like on your screenshot and when I click on the "categories" SliderItem it just showing me BetsFragment and it does not expand.

**Comment by Prototik** [comment]
... oh shit. Of course, I forgot about subitems :-/
Two ways:
- Replace ListView with ViewStubHolo (addon automatically replace it with needed view) (but then you lose support of layout attrs) (but you can avoid this by using FrameLayout)
- Replace ListView with ExpandableListView

**Comment by Alexwijn** [comment]
Thanks for the quick reply. I tried both methods. The first method does not change anything and the second only adds an arrow at the start of the button. I only change ListView at my menu.xml, you mean that one right?

**Comment by scott2014** [comment]
After read this,l solved my problem too,thank you so much
